src/actions/__stories__/Button.docs.mdx

```diff
@@ -1,11 +1,18 @@
-import { ArgsTable } from "@storybook/addon-docs"
+import { Meta } from "@storybook/addon-docs"
+import { ArgTypes, Title, Subtitle, Description } from "@storybook/blocks"
 import Button from "../Button"
+import * as stories from "./Button.stories.tsx"
 
-# &lt;Button /&gt;
+<Meta of={stories} />
+
+<Title />
+<Subtitle>Button components are interactive elements that trigger an action when clicked.</Subtitle>
```

Comment:
So…this looks really nice in the new Storybook theme, but it'll also show up in the Props embeds in zeroheight. I guess that's fine, but something to keep in mind so it's not too repetitive when viewing in zeroheight.